.env.testing

```diff
@@ -0,0 +1,18 @@
+APP_NAME=Laravel
+APP_ENV=local
+APP_KEY=base64:YgMNJRoag2qPK1YEeZDMHSnIQrvD8NLNlQKLpAaS6VQ=
+
+STABILITY_API_KEY=sk-ec75O5C6eVCz6PLPdESpv3h4fftDk03JYjJalDBg2rjsLdtK
```

Comment:
Not a good idea to leave your secret key in public.

Reply:
@geosem42 oops :P. I'll update these from my end and ensure they stay out of the repo.